Index: qemu-3.0.0/linux-user/syscall.c
===================================================================
--- qemu-3.0.0.orig/linux-user/syscall.c
+++ qemu-3.0.0/linux-user/syscall.c
@@ -250,15 +250,7 @@ static type name (type1 arg1,type2 arg2,
 #define TARGET_NR__llseek TARGET_NR_llseek
 #endif
 
-#ifdef __NR_gettid
 _syscall0(int, gettid)
-#else
-/* This is a replacement for the host gettid() and must return a host
-   errno. */
-static int gettid(void) {
-    return -ENOSYS;
-}
-#endif
 
 /* For the 64-bit guest on 32-bit host case we must emulate
  * getdents using getdents64, because otherwise the host
