Index: xcb-proto-1.13/xcbgen/align.py
===================================================================
--- xcb-proto-1.13.orig/xcbgen/align.py
+++ xcb-proto-1.13/xcbgen/align.py
@@ -2,7 +2,13 @@
 This module contains helper classes for alignment arithmetic and checks
 '''
 
-from fractions import gcd
+from sys import version_info
+
+if version_info[:2] >= (3, 5):
+    from math import gcd
+else:
+    from fractions import gcd
+
 
 class Alignment(object):
 
