Index: syslinux-6.03/com32/gplinclude/memory.h
===================================================================
--- syslinux-6.03.orig/com32/gplinclude/memory.h
+++ syslinux-6.03/com32/gplinclude/memory.h
@@ -37,8 +37,6 @@ struct e820entry {
     uint64_t type;		/* type of memory segment */
 } __attribute__ ((packed));
 
-const char *const e820_types[5];
-
 void get_type(int, char *, int);
 void detect_memory_e820(struct e820entry *desc, int size_map, int *size_found);
 int detect_memory_e801(int *, int *);
